cyral/data_source_cyral_sidecar_cft_template_test.go

```diff
@@ -10,7 +9,7 @@ import (
 
 var cftSidecarConfig SidecarData = SidecarData{
 	Name: "sidecar-test",
-	Tags: []string{"tag1", "tag2"},
+	Tags: []string{"test"},
```

Comment:
since we have all tags as key value pairs, we should use similar example. "somekey":"somevalue"